public/review.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class Foo {
+  public function __construct($bar) {
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected 4 spaces, found 2
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->